rpc/session.go

```diff
@@ -0,0 +1,21 @@
+package rpc
+
+import (
+	"net"
+	"net/rpc"
+	"net/rpc/jsonrpc"
+)
+
+type Session struct {
+	*rpc.Client
+}
+
+func NewSession(conn net.Conn) *Session {
```

Comment:
**[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
exported function NewSession should have comment or be unexported